src/Sql/Ddl/AlterTable.php

```diff
@@ -54,117 +46,111 @@ class AlterTable extends AbstractSql implements SqlInterface
     protected $specifications = [
         self::TABLE => "ALTER TABLE %1\$s\n",
         self::ADD_COLUMNS  => [
-            "%1\$s" => [
-                [1 => "ADD COLUMN %1\$s,\n", 'combinedby' => ""]
+            '%1$s' => [
+                [1 => "ADD COLUMN %1\$s,\n", 'combinedby' => '']
             ]
         ],
         self::CHANGE_COLUMNS  => [
-            "%1\$s" => [
-                [2 => "CHANGE COLUMN %1\$s %2\$s,\n", 'combinedby' => ""],
+            '%1$s' => [
+                [2 => "CHANGE COLUMN %1\$s %2\$s,\n", 'combinedby' => ''],
             ]
         ],
         self::DROP_COLUMNS  => [
-            "%1\$s" => [
-                [1 => "DROP COLUMN %1\$s,\n", 'combinedby' => ""],
+            '%1$s' => [
+                [1 => "DROP COLUMN %1\$s,\n", 'combinedby' => ''],
             ]
         ],
         self::ADD_CONSTRAINTS  => [
-            "%1\$s" => [
-                [1 => "ADD %1\$s,\n", 'combinedby' => ""],
+            '%1$s' => [
+                [1 => "ADD %1\$s,\n", 'combinedby' => ''],
             ]
         ],
         self::DROP_CONSTRAINTS  => [
-            "%1\$s" => [
-                [1 => "DROP CONSTRAINT %1\$s,\n", 'combinedby' => ""],
+            '%1$s' => [
+                [1 => "DROP CONSTRAINT %1\$s,\n", 'combinedby' => ''],
             ]
         ]
     ];
 
-    /**
-     * @var string
-     */
+    /** @var string */
     protected $table = '';
 
     /**
      * @param string|TableIdentifier $table
      */
     public function __construct($table = '')
     {
-        ($table) ? $this->setTable($table) : null;
+        $table ? $this->setTable($table) : null;
     }
 
-    /**
-     * @param  string $name
-     * @return self Provides a fluent interface
-     */
-    public function setTable($name)
+    public function setTable(string $name) : self
     {
         $this->table = $name;
 
         return $this;
     }
 
     /**
-     * @param  Column\ColumnInterface $column
-     * @return self Provides a fluent interface
+     * @param ColumnInterface $column
+     * @return self
      */
-    public function addColumn(Column\ColumnInterface $column)
+    public function addColumn(ColumnInterface $column) : self
     {
         $this->addColumns[] = $column;
 
         return $this;
     }
 
     /**
-     * @param  string $name
-     * @param  Column\ColumnInterface $column
-     * @return self Provides a fluent interface
+     * @param string $name
+     * @param ColumnInterface $column
+     * @return self
      */
-    public function changeColumn($name, Column\ColumnInterface $column)
+    public function changeColumn(string $name, ColumnInterface $column) : self
     {
         $this->changeColumns[$name] = $column;
 
         return $this;
     }
 
     /**
-     * @param  string $name
-     * @return self Provides a fluent interface
+     * @param string $name
+     * @return self
```

Comment:
we can drop the PHPDocs as it doesn't contain any additional information than method signature (see ZCS v2)